feature1/one-more-file.txt

```diff
@@ -0,0 +1,2 @@
+this file was also added in feature1 branch
```

Comment:
good to add this line of code

Reply:
thanks for your comment mouradzoghbi2020